scripts/monitoring/cron-send-docker-oc-versions.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python
+'''
+  Send openshift and docker versions with miq_metric tag to metric_sender
+
+  Example:
+  ./cron-send-docker-oc-versions.py -u oc_username -p oc_password
+'''
+# Disabling invalid-name because pylint doesn't like the naming conention we have.
+# pylint: disable=invalid-name,import-error
+
+from docker import AutoVersionClient
+import argparse
+from openshift_tools.monitoring.metric_sender import MetricSender
+import os
+import requests
+
+
+def parse_args():
+    '''Parse the arguments for this script'''
+    parser = argparse.ArgumentParser(description="Tool to send docker and openshift versions")
+    parser.add_argument('-u', '--username', action="store", help="openshift username")
+    parser.add_argument('-p', '--password', action="store", help="openshift password")
+    parser.add_argument('-d', '--debug', default=False, action="store_true", help="debug mode")
+    parser.add_argument('-v', '--verbose', default=False, action="store_true", help="Verbose?")
+    parser.add_argument('-s', '--host', help='specify openshift host name')
+
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    '''get docker and openshift versions and send to metric sender
+    '''
+
+    args = parse_args()
+
+    mts = MetricSender(verbose=args.verbose, debug=args.debug)
+
+    # Get docker version
+    cli = AutoVersionClient(base_url='unix://var/run/docker.sock', timeout=120)
+    docker_version = cli.version()["Version"]
+    mts.add_metric({"docker.version": docker_version}, key_tags={'miq_metric': 'true'})
+
+    # Get openshift version
+    if args.username:
+        creds = (args.username, args.password)
+    else:
+        creds = None
+
+    hostname = args.host or os.environ['HOSTNAME']
+    url = "https://{0}:8443/version/openshift".format(hostname)
```

Comment:
I'd prefer not to hard code the entire URL here.

I'm fine with `/varsion/openshift`, but I'd prefer it if the rest of the URL came from a config file, or passed on the cli.

Here's how we did it to gather other openshift master data:
https://github.com/openshift/openshift-tools/blob/stg/scripts/monitoring/cron-send-os-master-metrics.py#L64-L68

You don't have to do it that way specifically, but that's an example.

The other thing you may want to consider is using our OpenshiftRestApi class like the example above shows. It's a pretty nice way to interact with the master api.

Here's an example:
https://github.com/openshift/openshift-tools/blob/stg/scripts/monitoring/cron-send-os-master-metrics.py#L160

You could do something like this (untested):
```python
response = ora.get('/version/openshift')
```